localization/languages/pt-BR.json

```diff
@@ -31,89 +31,89 @@
         /* custom !bangs
         these are some of the items that show up when you press ! in the searchbar.
         Each one of these strings describes what the command will do when you run it. */
-        "showMoreBangs": null, //missing translation
+        "showMoreBangs": "Mostrar mais...",
```

Comment:
Can you remove the ellipsis from this?

Reply:
[...], indicates omission of something, in the case "show more (bangs)"